backend/src/bundles/users/user.repository.ts

```diff
@@ -21,6 +22,21 @@ class UserRepository implements Omit<IRepository, 'update' | 'delete'> {
         return UserEntity.initialize(user);
     }
 
+    public async findUserInfo(
```

Comment:
```suggestion
    public async getCurrentUserDetails(
```

Reply:
Fixed in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/300/commits/2bb73852884cc8f060d7f91bffc5474f3f439b69](url)